frontend/public/components/graphs/tooltip.tsx

```diff
@@ -0,0 +1,296 @@
+// Mostly just a copy-paste from patternfly.
```

Comment:
It's too bad you needed to do this.

I realize you need to be able to calculate the total, but are most of the other changes cosmetic? How different does the tooltip look without your changes?

Reply:
most of the changes are cosmetic. I've created issue at PF https://github.com/patternfly/patternfly-react/issues/4589 where I described all I needed to change